examples/benchmark_xgboost.py

```diff
@@ -0,0 +1,1100 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
```

Comment:
@haozhenWu Ah I see! Basically I suggest to use parser in command-line operations, like `python benchmark.py -m xgb_classifier -m xgb_regression` could help you do the same thing. Adding another script maybe a bit redundant for the deepchem package, would you mind removing it?